chrome/navigation.json

```diff
@@ -0,0 +1,736 @@
+{
+    "landingPage": [
+        {
+            "id": "application-services",
+            "title": "Application services",
+            "pathname": "/application-services"
+        },
+        {
+            "id": "openshift",
+            "title": "OpenShift",
+            "pathname": "/openshift"
+        },
+        {
+            "id": "rhel",
+            "title": "Red Hat Enterprise Linux",
+            "pathname": "/insights"
+        },
+        {
+            "id": "ansbile",
+            "title": "Ansible Automation Platform",
+            "pathname": "/ansible"
+        }
+    ],
+    "userPreferences": [
```

Comment:
Is there any reason why `userPreferences` and `settings` is lifted from `leftNav`? I understand the `landingPage` navigation. But if we are speaking just about navigation we shouldn't treat these any differently than other parts.

Speaking about the content, there should be a special file just to populate `userPreferences` content (mostly copy of what is already in the [config.json](https://github.com/RedHatInsights/user-preferences-frontend/blob/master/src/config/config.json))

Reply:
Stuff will be plit into different files.